spec/index.bs

```diff
@@ -3149,36 +3149,36 @@ dictionary GPUBindGroupLayoutEntry {
 
     : <dfn>buffer</dfn>
     ::
-        When not `undefined`, indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}}
-        is {{GPUBufferBinding}}.
+        Indicates the [=binding resource type=] for this {{GPUBindGroupLayoutEntry}} is
+        {{GPUBufferBinding}} if it [=map/exists=].
```

Comment:
The antecedent of "it" is unclear. It sounds like the antecedent is "GPUBufferBinding" but really it's "buffer".

What do you think of something like "If [=map/exists|provided=], indicates ..."

Alternatively "If this member [=map/exists=], it indicates ..."